jvm/src/main/scala/io/glutenproject/expression/UnaryOperatorTransformer.scala

```diff
@@ -94,7 +94,34 @@ class AbsTransformer(child: Expression, original: Expression)
     with ExpressionTransformer
     with Logging {
   override def doTransform(args: java.lang.Object): ExpressionNode = {
-    throw new UnsupportedOperationException("Not supported: Abs.")
+    val child_node = child.asInstanceOf[ExpressionTransformer].doTransform(args)
```

Comment:
Please use Camel-Case name for child_node.

Reply:
Done